### PR TITLE
Add OfisPublik Dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

File: datasets/ofis_publik/ofis_publik.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+_DESCRIPTION = """\
+Texts from the Ofis Publik ar Brezhoneg (Breton Language Board) provided by Francis Tyers
+2 languages, total number of files: 278
+total number of tokens: 2.12M
+total number of sentence fragments: 0.13M
+"""
+_HOMEPAGE_URL = "http://opus.nlpl.eu/OfisPublik.php"
+_CITATION = """\
+Tiedemann, J., 2012, Parallel Data, Tools and Interfaces in OPUS. In Proceedings of the 8th International Conference on Language Resources and Evaluation (LREC 2012) and
+Tyers, F. M. (2009) "Rule-based augmentation of training data in Breton-French statistical machine translation ". Proceedings of the 13th Annual Conference of the European Association of Machine Translation, EAMT09. pp. 213--218
```
Comment (lhoestq) [inline datasets/ofis_publik/ofis_publik.py:31]: can you use the bibtex citation ?

**Review by lhoestq — COMMENTED** [review]
thanks :) can you just update the citation ?

**Comment by abhishekkrthakur** [comment]
@lhoestq fixed :)

**Review by lhoestq — APPROVED** [review]
Thank you !

**Pull request merged**